Server/JokesOnYou.Web.Api/Models/Tag.cs

```diff
@@ -10,6 +11,7 @@ public class Tag
         public int Id { get; set; }
         public string Name { get; set; }
         public DateTime Created { get; set;  } = DateTime.Now;
+        [NotMapped]
```

Comment:
If this is notmapped, what is this list for since its data wont be persisted?

Reply:
i honestly have no idea. this came in when I merged dev into this branch. Had to use the NotMapped attribute otherwise it wouldnt run. 

I'm not sure how table joins work in code models.